### ISSUE TITLE
Exploration / Veteran Request: Alternate ways to add a provider, etc. in picker lists

### ISSUE THREAD

**Issue by brea11y** [department-of-veterans-affairs/va-mobile-app#9440]
In the BVA / Section 508 monthly meeting on August 21, 2024, one of the participants talked about her current experience with selecting a provider, category, etc. when sending a new secure message. This participant has roughly 25 providers with VA and is a screenreader user. To select a provider, she has to activate the picker list, then swipe through the list of providers, tap to select the provider, then swipe all the way back up to "done", the tap that to actually select and input the provider into the field. She asked if it would be possible to have the provider selected when she taps the name initially vs. being forced to navigate all the way back to "done".

I do not think the solution is as simple as just removing the "done" link and could be worth exploring other options. Other participants on the call asked if there was a way to narrow the picker list down by letter, some asked if we offered a way to search for a provider name, etc. I would recommend spending some time to see if there is a way that we could make the picker lists (especially for providers) simpler, faster, and easier to use for screenreader users.

**Comment by htcollier** [comment]
In an custom picker overlay where the user is limited to a single choice (like the one mentioned above), it's not necessary to have a Done button—selecting an item from the list could close the overlay and input the choice into the field (Cancel closes the overlay without inputting anything). "Done" is only necessary when the user is allowed to select multiple items from a list for input.

<img width="449" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBMG1FQmc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--22627748c833aeecaa889942b7aa05c90b7d7ea7/IMG_6341.PNG" alt="IMG_6341.PNG" />

**Comment by brea11y** [comment]
@htcollier – I agree. The Veteran actually just asked if she could tap the provider she wanted and not have to tap "done" at all. In the meeting, Martha did bring up the risk of making a mistake by tapping the wrong provider, but the Veteran said "That's on me, then. That's my mistake. I should have double checked before I sent the message."

**Comment by brea11y** [comment]
Ticket has been reviewed and is still valid.
